src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php

```diff
@@ -19,6 +19,7 @@
 use Symfony\Component\Validator\Constraints\NotBlank;
 use Symfony\Component\Validator\Constraints\NotNull;
 use Symfony\Component\Validator\Constraints\Range;
+use Symfony\Component\Validator\Constraints\IsTrue;
 use Symfony\Component\Validator\Constraints\True;
```

Comment:
Yes, let's try that. Otherwise we can wrap the use statement in an if block
and test for the PHP version.

2015-05-10 13:28 GMT+02:00 Abdellatif Ait boudad <notifications@github.com>:

> In
> src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php
> <https://github.com/symfony/symfony/pull/14228#discussion_r30001164>:
>
> > @@ -19,6 +19,7 @@
> >  use Symfony\Component\Validator\Constraints\NotBlank;
> >  use Symfony\Component\Validator\Constraints\NotNull;
> >  use Symfony\Component\Validator\Constraints\Range;
> > +use Symfony\Component\Validator\Constraints\IsTrue;
> >  use Symfony\Component\Validator\Constraints\True;
>
> can you try with use ..\True as LegacyTrue; or is there another way to
> avoid Fatal error ?
> PHP Fatal error: Cannot use Symfony\Component\Validator\Constraints\True
> as True because 'True' is a special class name in
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/symfony/symfony/pull/14228/files#r30001164>.
>